packages/manager/apps/zimbra/src/data/api/order/api.ts

```diff
@@ -1,6 +1,7 @@
 import { v6 } from '@ovh-ux/manager-core-api';
 import { OvhSubsidiary } from '@ovh-ux/manager-react-components';
-import { order } from './type';
+import { Order } from '@ovh-ux/manager-module-order';
+import { order, UpgradeServiceOrderParamsType } from './type';
```

Comment:
```suggestion
import { order, UpgradeServiceOrderParamsType } from '@/type';
```
i think it's better to use the alias path